src/local/config/__init__.py

```diff
@@ -0,0 +1,120 @@
+"""
+Configuration handling
+"""
+from __future__ import annotations
```

Comment:
Do we need given we are targetting 3.11+?

Reply:
I thought so too but it looks like it isn't rolled out yet https://docs.python.org/3/library/__future__.html#id1